### DISCUSSION TITLE
Question about submission quota

### DISCUSSION THREAD

**Discussion by hongyingyue** [PaddlePaddle/PaddleSpatial#287] [BaiduKDDCup2022]
Dear organizers,

I have some questions about submission quota. 

- If I submit 3 times between 20:00 16th (Beijing Time) and 23:59 in 16th (Beijing Time), do I still have 3 times between 00:00 19th (Beijing Time) and 20:00 19th (Beijing Time) , or because I have used 9 times, I can't submit again?
- If I submit 1 time in 16th 4 hours, how many can I have in 19th? Is it 2 times or 3 times
- Or it doesn't matter how many times in 16th, everyone has 3 times in 19th (Beijing Time) ?

Thanks in advance

**Comment by ramber1836** [comment]
> Dear organizers,
> 
> I have some questions about submission quota.
> 
> * If I submit 3 times between 20:00 16th (Beijing Time) and 23:59 in 16th (Beijing Time), do I still have 3 times between 00:00 19th (Beijing Time) and 20:00 19th (Beijing Time) , or because I have used 9 times, I can't submit again?

Yes, you still have 3 times between 00:00 and 20:00 on July 19 (Beijing Time). 

> * If I submit 1 time in 16th 4 hours, how many can I have in 19th? Is it 2 times or 3 times
> * Or it doesn't matter how many times in 16th, everyone has 3 times in 19th (Beijing Time) ?
> 
> Thanks in advance

Everyone has 3 times on July 19 (Beijing Time).
